src/app/pages/quickMaps/components/quickMapsHeader.component/quickMapsHeader.component.html

```diff
@@ -1,10 +1,46 @@
+<div class="container">
+  <div class="item-center">
+    <div *ngIf="(quickMapsService.micronutrientObs | async).isDiet">
```

Comment:
It can be diet and biomarker.  Might be better to test the selection using something like this:
quickMapsService.measure === measuretypeEnum.DIET

Reply:
How can it be both? The current toggle only allows Diet or Biomarker. What combination allows both?

Reply:
![image](https://user-images.githubusercontent.com/59727084/116244880-40281600-a760-11eb-81c1-5a89c7f6e066.png)
Iodine allows both so both of the tests you're doing will be true and you get both titles:
https://preview456--micronutrientsupport-tool.netlify.app/quick-maps/diet/baseline?country-id=MWI&mnd-id=I&measure=diet&data-level=household&age-gender-group=all
If you test the which measure has been selected (quickMapsService.measure) rather than what measures are available on the micronutrient you can achieve what you want to :-)

Reply:
ah ok will make the change cheers

Reply:
@jon571 this should be fixed now